src/glue/job.ts

```diff
@@ -52,45 +52,174 @@ export interface ContinuousLoggingProps {
   readonly defaultArguments?: {[key: string]: string};
+  /**
+   * A description of the job. 
+   */
   readonly description?: string;
+  /**
+   * Boolean value for whether to enable Profiling Metrics 
+   */
   readonly enableProfilingMetrics?: boolean;
+  /**
+   * Executable properties for the Job 
+   */
   readonly executable: JobExecutable;
+  /**
+   * The number of AWS Glue data processing units (DPUs) that can be allocated when this job runs. A DPU is a relative measure of processing power that consists of 4 vCPUs of compute capacity and 16 GB of memory. 
+   * 
+   * Do not set Max Capacity if using WorkerType and NumberOfWorkers.
+   * 
+   * The value that can be allocated for MaxCapacity depends on whether you are running a Python shell job or an Apache Spark ETL job:
+   * 
+   *    - When you specify a Python shell job (JobCommand.Name="pythonshell"), you can allocate either 0.0625 or 1 DPU. The default is 0.0625 DPU.
+   * 
+   *    - When you specify an Apache Spark ETL job (JobCommand.Name="glueetl"), you can allocate from 2 to 100 DPUs. The default is 10 DPUs. This job type cannot have a fractional DPU allocation.
+   * 
+   */
   readonly maxCapacity?: number;
+  /**
+   * Maximum number of concurrent executions 
+   * 
+   * @see [AWS::Glue::Job ExecutionProperty](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-job-executionproperty.html)
+   */
   readonly maxConcurrentRuns?: number;
+  /**
+   * The maximum number of times to retry this job after a JobRun fails. 
+   * 
+   * @see [AWS::Glue::Job](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-job.html#cfn-glue-job-maxretries)
+   */
   readonly maxRetries?: number;
+  /**
+   * A name for the Job 
+   */
   readonly name?: string;
+  /**
+   * After a job run starts, the number of minutes to wait before sending a job run delay notification.
+   * 
+   * @see [AWS::Glue::Job NotificationProperty](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-job-notificationproperty.html)
+   */
   readonly notifyDelayAfter?: Duration;
+  /**
+   * The name or Amazon Resource Name (ARN) of the IAM role associated with this job.
+   */
   readonly role?: IRole;
+  /**
+   * The Security Configuration object to be applied to the Job 
+   */
   readonly securityConfiguration?: SecurityConfiguration;
+  /**
+   * The job timeout in minutes. This is the maximum time that a job run can consume resources before it is terminated and enters TIMEOUT status. The default is 2,880 minutes (48 hours).
+   * 
+   * @see [AWS::Glue::Job](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-job.html#cfn-glue-job-timeout)
+   */
   readonly timeout?: Duration;
+  /**
+   * The number of worker available the Job 
+   */
   readonly workerCount?: number;
+  /**
+   * The type of predefined worker that is allocated when a job runs. Accepts a value of Standard, G.1X, or G.2X.
```

Comment:
Rather than specifying what is accepted here, that documentation should be moved to the referenced enum.

Reply:
Enum was already documented with this info. Removed comments from here.